.goreleaser.yaml

```diff
@@ -0,0 +1,33 @@
+# yaml-language-server: $schema=https://goreleaser.com/static/schema.json
+project_name: imagebuilder
+before:
+  hooks:
+    - go mod tidy
+    - go generate ./...
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    ignore:
+      - goos: windows
+        goarch: arm64
+archives:
```

Comment:
This will currently build binaries only, I think. For building and pushing an image you'd need something like [this](https://github.com/GoTestTools/gotestfmt/blob/main/.goreleaser.yml#L20-L25).